<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAlderImageWidget</class>
 <widget class="QWidget" name="QAlderImageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="horizontalFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="flipVerticalButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>flip image vertically</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources/QAlderGenericWidgets.qrc">
          <normaloff>:/icons/flipVicon.png</normaloff>:/icons/flipVicon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="flipHorizontalButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>flip image horizontally</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources/QAlderGenericWidgets.qrc">
          <normaloff>:/icons/flipHicon.png</normaloff>:/icons/flipHicon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="rotateCWButton">
        <property name="toolTip">
         <string>rotate clockwise 90 degrees</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources/QAlderGenericWidgets.qrc">
          <normaloff>:/icons/cwicon.png</normaloff>:/icons/cwicon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="rotateCCWButton">
        <property name="toolTip">
         <string>rotate counter clockwise 90 degrees</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources/QAlderGenericWidgets.qrc">
          <normaloff>:/icons/ccwicon.png</normaloff>:/icons/ccwicon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="interpolationToggleButton">
        <property name="toolTip">
         <string>linear / nearest neighbor interpolation </string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources/QAlderGenericWidgets.qrc">
          <normaloff>:/icons/linearicon.png</normaloff>:/icons/linearicon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="invertButton">
        <property name="toolTip">
         <string>invert window level</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources/QAlderGenericWidgets.qrc">
          <normaloff>:/icons/inverticon.png</normaloff>:/icons/inverticon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="backgroundButton">
        <property name="toolTip">
         <string>select background color</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="foregroundButton">
        <property name="toolTip">
         <string>select foreground color</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QAlderSliceView" name="sliceView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QAlderFramePlayerWidget" name="framePlayerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAlderSliceView</class>
   <extends>QWidget</extends>
   <header>QAlderSliceView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QAlderFramePlayerWidget</class>
   <extends>QWidget</extends>
   <header>QAlderFramePlayerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/QAlderGenericWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
